lib/vmdb/config/activator.rb

```diff
@@ -22,6 +22,10 @@ def log(data)
         Vmdb::Loggers.apply_config(data)
       end
 
+      def ntp(_data)
+        MiqServer.my_server.ntp_reload_queue unless MiqServer.my_server.nil? rescue nil
```

Comment:
I really don't like the conditional/rescue at the end, but it matches the other activator methods, and I'm not interested in solving why it's written that way right now